kpops/component_handlers/kafka_connect/kafka_connect_handler.py

```diff
@@ -36,12 +41,22 @@ async def create_connector(
         if dry_run:
             await self.__dry_run_connector_creation(connector_config)
         else:
+            connector_name = connector_config.name
             try:
-                await self._connect_wrapper.get_connector(connector_config.name)
+                await self._connect_wrapper.get_connector(connector_name)
                 await self._connect_wrapper.update_connector_config(connector_config)
+                match state:
```

Comment:
I think we should pause/stop before updating the connector and resume after updating the connector. That way we do not accidentally write data with a new config when connector should be paused/stopped and we don't write data with a new config when connector should be resumed

Reply:
I can change the order but calling pause/stop on a connector is async anyways, so the connector will not transition into paused/stopped state in time for the update config call

Reply:
I would change it anyways and hope that connect handles the requests in order

Reply:
right! fixed